x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_settings.tsx

```diff
@@ -18,9 +18,6 @@ export function LayerSettings({
 }) {
   if (!activeVisualization.LayerHeaderComponent) {
     const description = activeVisualization.getDescription(layerConfigProps.state);
-    if (!description) {
```

Comment:
this is already validated by typescript so it's safe to remove

Reply:
Awesome. Type validation costs less than test validation.